README.md

```diff
@@ -11,8 +11,22 @@ Once you've cloned the repo, run
 
 ### Publishing changes to NPM
 
-You need a @guardian scoped NPM user account to be able to publish changes. You then will need to login to this NPM account locally on your maching. Once authenticated, run:
-`yarn build && yarn publish` updating the version number as required. It will publish as `public` as The Guardian Organisation does not have private repos.
+Versioning and publishing for this package is managed through [changesets](https://github.com/changesets/changesets).
+
+If you make a change to the repo which merits a version bump, you should add a changeset
+to your PR:
+
+1. Run `yarn changeset`.
+2. The changeset CLI will prompt you for a semantic version level: `major`, `minor`, or `patch`.
+3. It will then prompt you for a description of the change introduced by your PR.
+
+When a PR with a changeset is merged to `main`, changesets will automatically open a new PR
+which, when merged, will automatically bump the version number using (following [semver conventions](https://semver.org/)),
+publish the new version to npm, and update the github changelog for this package.
+
+Changesets is a relatively new addition to this repo, so if you run into any difficulties using it, 
+please feel free to [open an issue](https://github.com/guardian/discussion-rendering/issues/new)
+in this repo.
```

Comment:
I think this should be a note, but separate from the 4 tasks above.

If a user does not complete step 4, their package will not have been bumped, which is very unlikely what they wanted.

```suggestion
> **Note**
> When a PR with a changeset is merged to `main`, changesets will automatically open a new PR
> which, when merged, will automatically bump the version number using (following [semver conventions](https://semver.org/)),
> publish the new version to npm, and update the github changelog for this package.

Changesets is a relatively new addition to this repo, so if you run into any difficulties using it, 
please feel free to [open an issue](https://github.com/guardian/discussion-rendering/issues/new)
in this repo.
```

Reply:
Yes that seems right. I was thinking of the fact that (iirc) changesets allows you to collect a number of changes together in a batch before publishing, but I think you're right that that won't be the typical workflow 👍